src/main/java/org/folio/rest/impl/DelayedTasksAPI.java

```diff
@@ -15,6 +19,9 @@
 
 public class DelayedTasksAPI extends AbstractEmail implements DelayedTask {
 
+  private static final String REQUEST_QUERY_SHOULD_RETRY = "shouldRetry=true";
+  private static final int DEFAULT_LIMIT = 1000;
```

Comment:
```suggestion
  private static final int DEFAULT_LIMIT = 100;
```

Use only when configuration in mod-config is missing